roles/dev_datastore/apache2/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- include_role:
+    name: common/install_dependencies
+
+- name: Enable mod_headers
```

Comment:
Different installations of `apache` will require different modules. Suggest extracting the list of required modules into a variable and using `{{ items }}`.